### DISCUSSION TITLE
i want my website to have a /test behind it for example my website is 7ib0.github.io but i want to make a 7ib0.github.io/test

### DISCUSSION THREAD

**Discussion by 7ib0** [community/community#132158] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried by making a file but that didnt work...
![test](https://github.com/user-attachments/assets/f0957012-0e4c-4f3c-a4ed-f44eea7cfd7a)

**Comment by 7ib0** [comment]
Oh and i forgot to say: i use vscode if that helps...

**Comment by SajanGhimire1** [comment]
@7ib0 
Rename index2.html to index.html in the test folder and push the changes to your GitHub repository. This will make 7ib0.github.io/test display your page correctly & push the update to GitHub so 7ib0.github.io/test displays properly.
